src/Network/Network/Cortex/HubVnetConnection/NewAzureRmHubVirtualNetworkConnectionCommand.cs

```diff
@@ -156,33 +156,22 @@ public override void Execute()
                 throw new PSArgumentException(string.Format(Properties.Resources.ChildResourceAlreadyPresentInResourceGroup, this.Name, this.ResourceGroupName, this.ParentResourceName));
             }
 
-            //// Get the virtual hub - this will throw not found if the resource does not exist
-            PSVirtualHub parentVirtualHub = this.GetVirtualHub(this.ResourceGroupName, this.ParentResourceName);
-            if (parentVirtualHub == null)
-            {
-                throw new PSArgumentException(Properties.Resources.ParentVirtualHubNotFound);
-            }
-
-            PSHubVirtualNetworkConnection hubVnetConnection = new PSHubVirtualNetworkConnection();
+            var hubVnetConnection = new MNM.HubVirtualNetworkConnection();
             hubVnetConnection.Name = this.Name;
+            // Fixme.. make it a boolean
```

Comment:
it is currenlty a switch parameter i suppose..

Reply:
Yes. I did not want to break existing scripts which specify the parameter & punish users who were actually using the option. The other suggestion I got on the powershell channel was to simply introduce a new parameter.